.changeset/three-sheep-sparkle.md

```diff
@@ -0,0 +1,21 @@
+---
+'@backstage/plugin-catalog-backend': minor
+---
+
+**BREAKING** Add authorized refresh service
```

Comment:
Let's bring the actual breaking change up to the front, which in this case would be that we now require `permissions` to be part of the `CatalogEnvironment`. Not sure we really need to call it out this loudly though tbh, since our recommended setup forwards the entire plugin environment and so there's no need need for any changes to the catalog setup.

Reply:
"Add authorized refresh service" isn't a breaking change. Let's either bring up the actual breaking change or in this case I think we could drop this line too, because any standard catalog setup won't need any change, and this is a `minor` bump anyway